### PR TITLE
Removed non-functional menu button

### LINKED ISSUE #237: [Bug] An empty Menu Button in the Mobile screens
## Description
When you switch to mobile screen, a menu button appears. This menu has no content and hence should be removed.

## Expectation
There should be no Menu button since there are no contents. Also, in the desktop screens, there are no menus(there is no navbar).
- [x ] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by Queen-codes**
## Fixes

- Fixes #237  by @SisiVero 

## Description
On mobile screens, there is a menu button that is not functional, clicking on that button does nothing to the website and has no content. This pull request removes that button.

## Tests
1. Open the CC's search website
2. Resize the browser to smaller screens or view on a mobile phone
3. Menu button should no longer be there.

## Screenshots
Problem(before)
Screenshot 2024-10-07 at 17 18 59

Solution(after)
Screenshot 2024-10-07 at 17 16 23

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!DOCTYPE html>
```
Comment (possumbilities) [inline src/index.html:1]: this is a change outside the scope of the PR please revert it :)

File: .DS_Store
Comment (possumbilities) [inline .DS_Store:1]: this file is not a part of these changes, please remove it.

File: src/.DS_Store
Comment (possumbilities) [inline src/.DS_Store:1]: this file is not a part of these changes, please remove it.

Comment (Queen-codes) [reply src/.DS_Store:1]: okay, thank you

**Comment by Queen-codes** [comment]
Thank you @possumbilities  I appreciate it. Looking forward to more contributions.

**Comment by possumbilities** [comment]
@Queen-codes If you comment on the original Issue I can assign it to you!

**Pull request merged**